logging/Differences_between_ODFE_and_OpenSearch.md

```diff
@@ -0,0 +1,97 @@
+**Do not** use the "move2opensearch" branch in the repository.
+
+## Differences
+
+### Products
+
+The unstructured data store and search back-end component, formerly Elasticsearch, is now OpenSearch. 
+
+The visualization and reporting application, formerly Kibana, is now OpenSearch Dashboards.
+
+### Helm Charts
+
+While OpenSearch is virtually identical to Open Distro for Elasticsearch, the Helm chart used to deploy OpenSearch is different. This required the following changes in the customization process: 
+
+* The names of the YAML files used to customize the deployment have changed.
+* The structure of these YAML files has changed. 
+
+See [Compiled Differences](#compiled_dif_table) for more information.
+
+### Deployment Topology
+
+The topology of the search back-end is different. 
+
+* The Open Distro for Elasticsearch search back-end is configured by default to use eight Kubernetes pods composed of two Elasticsearch client nodes, three Elasticsearch master nodes, and three Elasticsearch data nodes.
+* For the OpenSearch search back-end, the default configuration is three Kubernetes pods composed of three OpenSearch "multi-role" nodes.
+* The Java memory settings have increased from 1GB to 4GB for each OpenSearch node.
+
+### NodePort Accessibility
+
+If you had not configured access via Ingress, the Open Distro for Elasticsearch-based deployment script automatically made Kibana accessible via NodePort.
+
+The OpenSearch deployment script does not do this automatically. If you want to make OpenSearch Dashboards accessible via NodePort, you must set the environment variable `KB_KNOWN_NODEPORT_ENABLE` to `True` before running the `logging/bin/deploy_logging_opensearch.sh` script.
+
+Kibana was accessible from port 31033. OpenSearch Dashboards is accessible from port 31034.
+
+**Note:** NodePorts are not suitable for production deployments. NodePorts are intended only as a convenience for internal testing groups working in an OpenStack environment. 
+
+### Migrate an Existing Deployment
+
+To migrate an existing deployment of the log monitoring project, run the `./logging/bin/deploy_logging_opensearch.sh` script.
+If an existing deployment is detected in the namespace, that deployment
+is migrated to the OpenSearch-based deployment.
+
+### Remove OpenSearch-based Log Monitoring
+
+To remove log monitoring that uses OpenSearch instead of ODFE, run the `logging/bin/remove_logging_opensearch.sh` script. The new script supports all the same environment variables as the prior script.
+
+### Running Other Scripts
+
+Other than the scripts described in the [table](#compiled_dif_table), the remaining log monitoring  scripts work with either search back-end. 
+
+See [Compiled Differences](#compiled_dif_table) for the list of changed script file names.
+
+**Important:** During the experimental phase only, you must set the environment variable `LOG_SEARCH_BACKEND` to `OPENSEARCH` (in uppercase) when you run scripts with OpenSearch.
```

Comment:
Now that we're out of the experimental phase, strike this sentence.

Reply:
Removed.